cardano/base/templatetags/tags.py

```diff
@@ -13,3 +13,16 @@ def lookup_country_vouched(countries, key):
 def lookup_country_total(countries, key):
     q = max(countries, key=lambda d: (d['name'] == key))
     return q['total_diff']
+
+
+@register.filter
```

Comment:
The initial comment was to add a helper to return the correct ``glyphicon0`` and use it like
```
<span class="etc {% glyphicon|foo %} ... ></span>
```